src/SumoCoders/FrameworkCoreBundle/Form/Type/AbstractImageType.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+class AbstractImageType extends AbstractType
```

Comment:
👍

Op woensdag 11 mei 2016 heeft Jelmer Prins <notifications@github.com> het
volgende geschreven:

> In src/SumoCoders/FrameworkCoreBundle/Form/Type/AbstractImageType.php
> <https://github.com/sumocoders/Framework/pull/129#discussion_r62798520>:
>
> > @@ -0,0 +1,78 @@
> > +<?php
> > +
> > +namespace SumoCoders\FrameworkCoreBundle\Form\Type;
> > +
> > +use Symfony\Component\Form\AbstractType;
> > +use Symfony\Component\Form\FormBuilderInterface;
> > +use Symfony\Component\Form\FormInterface;
> > +use Symfony\Component\Form\FormView;
> > +use Symfony\Component\OptionsResolver\OptionsResolverInterface;
> > +
> > +class AbstractImageType extends AbstractType
>
> fair point, I'll refactor this :) this is the default implementation but
> since there are no abstract methods it can just be a class that gets
> extended
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/sumocoders/Framework/pull/129/files/3a03b36872f88a16bb78f3cda4c99a6100a5ffa5#r62798520>
>